dynamo/tools/velocyto_scvelo.py

```diff
@@ -212,6 +213,135 @@ def colormap_fun(x: np.ndarray) -> np.ndarray:
+                  "deterministic model. If this is not what you want, please change them manually.")
+        if "highly_variable_genes" in adata.var.columns:
+            adata.var["pass_basic_filter"] = adata.var.pop("highly_variable_genes")
+            adata.var["pass_basic_filter"] = [True if item == 'True' else False for item in
+                                              adata.var["pass_basic_filter"]]
+        if "spliced" in adata.layers.keys():
+            adata.layers["X_spliced"] = adata.layers.pop("spliced")
+        if "unspliced" in adata.layers.keys():
+            adata.layers["X_unspliced"] = adata.layers.pop("unspliced")
+        if "highly_variable" in adata.var.columns:
+            adata.var["use_for_pca"] = adata.var.pop("highly_variable")
+        if "recover_dynamics" in adata.uns.keys():
+            adata.uns.pop("recover_dynamics")
+            adata.uns["dynamics"] = {
+                "filter_gene_mode": None,
+                "t": None,
+                "group": None,
+                "X_data": None,
+                "X_fit_data": None,
+                "asspt_mRNA": "ss",
+                "experiment_type": "conventional",
+                "normalized": True,
+                "model": "deterministic",
+                "est_method": "ols",
+                "has_splicing": True,
+                "has_labeling": False,
+                "splicing_labeling": False,
+                "has_protein": False,
+                "use_smoothed": True,
+                "NTR_vel": False,
+                "log_unnormalized": True,
+                "fraction_for_deg": False,
+            }
+        adata.uns["pp"] = {
+            "has_splicing": True,
+            "has_labeling": False,
+            "splicing_labeling": False,
+            "has_protein": False,
+            "tkey": None,
+            "experiment_type": "conventional",
+            "X_norm_method": None,
+            "layers_norm_method": None,
+        }
+        if "Ms" in adata.layers.keys():
+            adata.layers["M_s"] = adata.layers.pop("Ms")
+        if "Mu" in adata.layers.keys():
+            adata.layers["M_u"] = adata.layers.pop("Mu")
+        if "velocity_u" in adata.layers.keys():
+            adata.layers["velocity_U"] = adata.layers.pop("velocity_u")
+        if "velocity" in adata.layers.keys():
+            adata.layers["velocity_S"] = adata.layers.pop("velocity")
+        if "fit_alpha" in adata.var.columns:
+            adata.var["alpha"] = adata.var.pop("fit_alpha")
+        if "fit_beta" in adata.var.columns:
+            adata.var["beta"] = adata.var.pop("fit_beta")
+        if "fit_gamma" in adata.var.columns:
+            adata.var["gamma"] = adata.var.pop("fit_gamma")
+        if "fit_r2" in adata.var.columns:
+            adata.var["gamma_r2"] = adata.var.pop("fit_r2")
+        if "fit_u0" in adata.var.columns:
+            adata.var["u0"] = adata.var.pop("fit_u0")
+        if "fit_s0" in adata.var.columns:
```

Comment:
add these to varm?

Reply:
I will do that after we merge the "Save velocity parameters to .varm instead .var" branch. Then we can utilize the helper function like get_vel_params() or update_vel_params().